app/controllers/help_controller.rb

```diff
@@ -1,4 +1,10 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    @data_chat = AppConfig.chat.enabled ? "enabled" : "disabled"
```

Comment:
> in fact it's less code you have to add in this PR.

In fact it's more code I have to remove. But if you think it's better, then let's do this.